code/restful-api/restful-api/Models/Cria.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+namespace RestfulApi.Models
+{
+    public class Cria
```

Comment:
A better name for this should be `OrganizadorUsuario`, no?

`Cria` is very vague.